superset-frontend/src/explore/components/ControlPanelsContainer.tsx

```diff
@@ -531,14 +588,23 @@ export const ControlPanelsContainer = (props: ControlPanelsContainerProps) => {
               placement="right"
               title={props.errorMessage}
             >
-              <i className="fa fa-exclamation-circle text-danger fa-lg" />
```

Comment:
I can't tell you how happy it makes me to see these fa-* icons fading away one by one :D